tests/slr_integration/Commerce/Controller_Test.php

```diff
@@ -313,4 +313,85 @@ public function test_filter_timer_token_object_id_entries( Closure $fixture ): v
 			$filtered_entries,
 		);
 	}
+	
+	public function test_stock_count_for_seated_tickets() {
+		$controller = $this->make_controller();
+		$controller->register();
+		$event_id = tribe_events()->set_args(
+			[
```

Comment:
Here, please include a scenario where we have 2 different tickets sharing the same seat type.